abci/client/grpc_client.go

```diff
@@ -314,20 +314,59 @@ func (cli *grpcClient) ApplySnapshotChunkAsync(
 	)
 }
 
-func (cli *grpcClient) PrepareProposalAsync(
+// NOTE: call is synchronous, use ctx to break early if needed
+func (cli *grpcClient) ExtendVoteAsync(
 	ctx context.Context,
-	params types.RequestPrepareProposal,
+	params types.RequestExtendVote,
 ) (*ReqRes, error) {
-
-	req := types.ToRequestPrepareProposal(params)
-	res, err := cli.client.PrepareProposal(ctx, req.GetPrepareProposal(), grpc.WaitForReady(true))
+	req := types.ToRequestExtendVote(params)
+	res, err := cli.client.ExtendVote(ctx, req.GetExtendVote(), grpc.WaitForReady(true))
 	if err != nil {
 		return nil, err
 	}
 	return cli.finishAsyncCall(
 		ctx,
 		req,
+		&types.Response{Value: &types.Response_ExtendVote{ExtendVote: res}},
+	)
+}
+
+// NOTE: call is synchronous, use ctx to break early if needed
```

Comment:
```suggestion
// VerifyVoteExtensionAsync ...
```

Same comment as above.